cpp/ql/src/Likely Bugs/Arithmetic/SignedOverflowCheck.qhelp

```diff
@@ -19,56 +19,36 @@ optimizing compiler.
 <p>
 Solutions to this problem can be thought of as falling into one of two
 categories: (1) rewrite the signed expression so that overflow cannot occur
-but the signedness remains, or (2) rewrite (or cast) the signed expression
-into unsigned form.
+but the signedness remains, or (2) change the variables and all their uses to
+be unsigned. The following cases all fall into the first category.
 </p>
```

Comment:
This paragraph is probably neater as:

```
<p>
Solutions to this problem can be thought of as falling into one of two
categories:
</p> 

<ol>
<li>Rewrite the signed expression so that overflow cannot occur
but the signedness remains.</li>
<li>Change the variables and all their uses to be unsigned.<li>
</ol>

<p>
The following cases all fall into the first category.
</p>
```

Reply:
Then perhaps those following cases should also be an `<ol>`?